test/parallel/test-process-dlopen-error-message-crash.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// This is a regression test for some scenarios in which node would pass
+// unsanitized user input to a printf-like formatting function when dlopen
+// fails, potentially crashing the process.
+
+require('../common');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const assert = require('assert');
+const fs = require('fs');
+
+// This error message should not be passed to a printf-like function.
+assert.throws(() => {
+  process.dlopen({ exports: {} }, 'foo-%s.node');
+}, {
+  name: 'Error',
+  code: 'ERR_DLOPEN_FAILED',
+  message: /foo-%s\.node/
+});
+
+// This error message should also not be passed to a printf-like function.
+const notBindingPath = './test/addons/not-a-binding/build/Release/binding.node';
+const strangeBindingPath = `${tmpdir.path}/binding-%s.node`;
+fs.copyFileSync(notBindingPath, strangeBindingPath);
```

Comment:
the source file doesn't seem to be existing on some platforms?

Reply:
Seems like it's platforms on which we skip `test/addons`, so `binding.node` is never built.